src/interpreter/Interpreter.ts

```diff
@@ -138,18 +150,36 @@ export class Interpreter {
   }
 
   /**
-   * Split up all models which should and use ref instead.
+   * Split up all model which should be and use $ref instead.
    * 
-   * @param model to ensure are split
-   * @param models which are already split
+   * @param model
    */
   ensureModelsAreSplit(model: CommonModel): void {
```

Comment:
```suggestion
  ensureModelsAreSplitted(model: CommonModel): void {
```

Reply:
This sounds weird? "splitted" are not a word 🙂

Reply:
😅 I was wondering too but I found the use of this word in some articles ... my mistake.